### PR TITLE
KAKUTEF7MINI camera control not-fix

### PR DESCRIPTION
**Pull request by gretel**
did try to make the camera control pin usable by syncing with betaflight.
but when the resource gets assigned motor 2,4 cant spin up.
looks like some time woe to me. maybe @Kaiowarez has a clue

### CODE REVIEW

**Comment by gretel** [comment]
dang i should have rebased

**Comment by Kaiowarez** [comment]
> did try to make the camera control pin usable by syncing with betaflight.
> but when the resource gets assigned motor 2,4 cant spin up.
> looks like some time woe to me. maybe @Kaiowarez has a clue

do you have a dump for BF version?
the mini builds from kakutef7 target, no camera control assigned need a timer dma and resource allocation for camera control to work

**Comment by gretel** [comment]
@Kaiowarez hang on a sec

**Comment by gretel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5342715&bid=17491715)

File: src/main/common/typeconversion.c [hunk 1]
```diff
@@ -188,7 +188,8 @@ char *ftoa(float x, char *floatString)
 
     dpLocation = strlen(intString2) - 3;
 
-    strncpy(floatString, intString2, dpLocation);
+    // https://github.com/emuflight/EmuFlight/pull/125#issuecomment-609475459
+    memcpy(floatString, intString2, dpLocation);
```
Comment (gretel) [inline src/main/common/typeconversion.c:192]: Codacy Issue found: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5342715)

File: README.md [hunk 2]
```diff
@@ -1,123 +1,71 @@
-![EmuFlight](docs/assets/images/EmuFlight.png)
+# EgoFlight
```
Comment (gretel) [inline README.md:1]: Codacy Issue found: [Could not find module `remark-lint-no-unneeded-full-reference-image`](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5342715)

**Comment by gretel** [comment]
@Kaiowarez rebased pr at https://github.com/emuflight/EmuFlight/pull/164

**Pull request closed without merging**